pkg/manifests/manifests.go

```diff
@@ -40,9 +38,9 @@ const (
 	ServingCertSecretAnnotation = "service.alpha.openshift.io/serving-cert-secret-name"
 
 	// OwningClusterIngressLabel should be applied to any objects "owned by" a
-	// clusteringress to aid in selection (especially in cases where an ownerref
+	// ingress controller to aid in selection (especially in cases where an ownerref
 	// can't be established due to namespace boundaries).
-	OwningClusterIngressLabel = "ingress.openshift.io/clusteringress"
+	OwningClusterIngressLabel = "ingress.operator.openshift.io/owning-ingresscontroller"
```

Comment:
Due to the name change, I believe `OwningClusterIngressLabel` should be updated to `OwningIngressControllerLabel`.